### PR TITLE
Remove "exotic" mentions in content

### PR DESCRIPTION
**Pull request by MattIPv4**
## Describe your changes

Request from the folks at Alveus, Alveus should be referred to as a wildlife sanctuary, not an exotic animal sanctuary.

## Notes for testing your change

All mentions of exotic removed, content still reads fine.

### CODE REVIEW

File: apps/website/src/pages/index.tsx [hunk 1]
```diff
@@ -239,11 +239,11 @@ const Home: NextPage = () => {
...
-                functions as an exotic animal sanctuary and as a virtual
-                education center facility to provide permanent homes to
-                non-releasable exotic animals. These animals function as
-                ambassadors, so viewers can watch their journeys, get to know
-                the animals, and gain an appreciation for their species.
+                functions as a wildlife sanctuary and as a virtual education
+                center facility to provide permanent homes to non-releasable
+                animal ambassadors. These animals function as ambassadors, so
+                viewers can watch their journeys, get to know the animals, and
+                gain an appreciation for their species.
```
Comment (MattIPv4) [inline apps/website/src/pages/index.tsx:246]: The sentences here feel a bit duplicative now, open to suggestions (or can leave it as-is, not a big deal)

**Review by pjeweb — APPROVED** [review]
Good enough, but I agree there is some redundancy. If I find the time before merging, I can try to rephrase, otherwise I'd merge as-is.

File: apps/website/src/pages/index.tsx (hunk 1 above)
Comment (JonasFranke) [reply apps/website/src/pages/index.tsx:246]: smth like Viewers are able to watch ambassador's journeys, get to know them and gain appreciation for their species?

Comment (MattIPv4) [reply apps/website/src/pages/index.tsx:246]: I'm going to leave this as-is for now so we can get this in, but feel free to open a PR if you do want to take a stab at rewording :)

**Pull request merged**